docs/setup_caen.md

```diff
@@ -82,17 +55,24 @@ Success. Logging you in...
 -bash-4.2$ 
 ```
 
-<div class="primer-spec-callout info icon-info" markdown="1">
-If you see an error like `Could not chdir to home directory /home/awdeorio: No such file or directory`, a common reason is that you don't have a home directory set up yet.  Request one at the [IFS Storage Request page](https://ifsprovisioning.its.umich.edu/ifs_storage/request). Note that it may take a few hours before your home directory is available across all CAEN systems. If you already have one, you should see "You already have IFS Storage!".
-
-If you're still having trouble accessing your account, see the [CAEN Help Desk](https://caen.engin.umich.edu/contact/).
-</div>
-
 Exit as soon as your test is successful.
 ```console
 $ exit
-Connection to login.engin.umich.edu closed.
+Connection to login-course.engin.umich.edu closed.
+```
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** If you are off campus, make sure you have connected to the [UM VPN](https://its.umich.edu/enterprise/wifi-networks/vpn/getting-started).
+</div>
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** If you see an error like this, request a home directory at the [IFS Storage Request page](https://ifsprovisioning.its.umich.edu/ifs_storage/request). It may take a few hours to take effect. If you already have a home directory one, you should see "You already have IFS Storage!".
```

Comment:
```suggestion
**Pitfall:** If you see an error like this, request a home directory at the [IFS Storage Request page](https://ifsprovisioning.its.umich.edu/ifs_storage/request). It may take a few hours to take effect. If you already have a home directory, you should see "You already have IFS Storage!".
```